src/utilities/add-remove.test.js

```diff
@@ -0,0 +1,203 @@
+/**
+ * @jest-environment jsdom
+ */
+const { expect, test } = require('@jest/globals');
+const Task = require('./task.js');
+const LocalStorage = require('./localStorage.js');
+
+describe('Test add', () => {
+  // Arrange --------------------------------------------------------------------------->
+  document.body.innerHTML = '<div>'
+     + '<input type="text" name="task" id="task" class="input-task" value="Hello">'
+     + '<ul class="container-list"></ul>'
+     + '</div>';
+  const event = {
+    keyCode: 13,
+  };
+
+  //   class LocalStorage {
+  //     static list = [];
+
+  //     static setDataLocalStorage(item) {
+  //       this.list = item;
+  //     }
+
+  //     static getDataLocalStorage() {
+  //       return this.list;
+  //     }
+  //   }
+  const localStorage = new LocalStorage();
+
+  const renderTaskDom = () => {
+    const listTasks = localStorage.getDataLocalStorage();
+    const container = document.querySelector('.container-list');
+    const input = document.querySelector('.input-task');
+    const task = document.createElement('li');
+    task.classList.add('item');
+    task.dataset.id = listTasks.length;
+    const checkBox = document.createElement('input');
+    checkBox.setAttribute('type', 'checkbox');
+    const label = document.createElement('label');
+    label.textContent = input.value;
+    const icon = document.createElement('i');
+    icon.classList.add('fas', 'fa-trash-alt', 'remove');
+
+    task.appendChild(checkBox);
+    task.appendChild(label);
+    task.appendChild(icon);
+    container.appendChild(task);
+  };
+
+  const checkElementsInTaskDOM = () => {
+    const task = document.querySelector('.item');
+    const childrenTask = [...task.children];
+    const checkBox = childrenTask[0];
+    const label = childrenTask[1];
+    const icon = childrenTask[2];
+    return [checkBox, label, icon, task];
+  };
+
+  const refreshDragDropTarget = jest.fn();
+  const refreshDescriptions = jest.fn();
+
+  // Act --------------------------------------------------------------------------->
+  const add = (event) => {
+    if (event.keyCode === 13) {
+      const listTasks = localStorage.getDataLocalStorage();
+      const input = document.querySelector('.input-task');
+      const inputValue = input.value;
+      const newTask = new Task(inputValue, false, listTasks.length);
+      listTasks.push(newTask);
+      localStorage.setDataLocalStorage(listTasks);
+      input.value = '';
+      renderTaskDom();
+      refreshDragDropTarget();
+      refreshDescriptions();
+      return listTasks;
+    }
+    return false;
+  };
```

Comment:
- Good job on the job done so far 👏.  I see that you have rewritten another add function here. The whole purpose of testing is to ensure that the functions responsible for doing specific tasks in our program work as expected. Hence you are expected to test the `add.js` function in your `module` dir.